LEGO1/act3state.h

```diff
@@ -0,0 +1,26 @@
+#ifndef ACT3STATE_H
+#define ACT3STATE_H
+
+#include "legostate.h"
+
+class Act3State : public LegoState
+{
+public:
+  Act3State();
+
+  // OFFSET: LEGO1 0x1000e300
+  inline virtual const char *GetClassName() const // vtable+0x0c
+  { 
+    return "Act3State";
+  }; 
+
+  // OFFSET: LEGO1 0x100d4fd8
```

Comment:
The vtable address was given.
```suggestion
  // OFFSET: LEGO1 0x1000e310
```